### PR TITLE
Add new manifest members.

### PR DESCRIPTION
**Pull request by jpmedley**
[display_override](https://www.chromestatus.com/feature/5728570678706176)
[note_taking and new_note_url](https://www.chromestatus.com/feature/5205972320518144)

### CODE REVIEW

**Comment by jpmedley** [comment]
@tomayac Please look this over.

File: html/manifest/display_override.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      "display_override": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/display_override",
+          "spec_url": "https://wicg.github.io/manifest-incubations/#display_override-member",
+          "support": {
+            "chrome": {
+              "version_added": "89"
+            },
+            "chrome_android": {
+              "version_added": false
```
Comment (tomayac) [inline html/manifest/display_override.json:13]:
```suggestion
              "version_added": "89"
```

This rolled out for Android as well according to [crbug/1092667](https://bugs.chromium.org/p/chromium/issues/detail?id=1092667). While window controls overlay and tabbed application mode don't make sense on Android, you can still use it with the traditional values like `minimal-ui` if you want to customize your fallback chain.

File: html/manifest/display_override.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": null
```
Comment (tomayac) [inline html/manifest/display_override.json:28]:
```suggestion
              "version_added": "75"
```

According to the [release notes](https://help.opera.com/en/opera-version-history/#:~:text=Version%20history-,Opera%2075,Chromium%2089,-JavaScript%20engine).

File: html/manifest/display_override.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            },
+            "opera_android": {
+              "version_added": false
```
Comment (tomayac) [inline html/manifest/display_override.json:31]:
```suggestion
              "version_added": "75"
```

According to the [release notes](https://help.opera.com/en/opera-version-history/#:~:text=Version%20history-,Opera%2075,Chromium%2089,-JavaScript%20engine).

File: html/manifest/display_override.json [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": false
```
Comment (tomayac) [inline html/manifest/display_override.json:40]: The latest Chromium engine the [release notes](https://developer.samsung.com/internet/release-note.html) mention is still Chromium 87.4280, so `false` might actually be correct. @torgo to verify.

**Review by tomayac — COMMENTED** [review]
The nesting is as follows:

```json
{
  "note_taking": {
    "new_note_url": "/new_note.html"
  }
}
```

Does the URL structure not reflect this? I don't know the answer, but at the moment it looks like they're flat next to each other.

**Comment by jpmedley** [comment]
> The nesting is as follows:
> 
> ```json
> {
>   "note_taking": {
>     "new_note_url": "/new_note.html"
>   }
> }
> ```
> 
> Does the URL structure not reflect this? I don't know the answer, but at the moment it looks like they're flat next to each other.

It looks like they're ignoring all the nesting. I don't think we can deal with the issue in this PR. @Elchi3 What was the thinking behind this?

**Comment by jpmedley** [comment]
Brain not working. I would just make one the child of the other in the same file (just like all the other BCD.)

**Comment by jpmedley** [comment]
Now it's nested.

File: html/manifest/note_taking.json [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "new_note_url": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/new_note_url",
```
Comment (Elchi3) [inline html/manifest/note_taking.json:54]: I guess this should either be a sub page

```suggestion
            "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/note_taking/new_note_url",
```

Or a section on the note_taking page.

```suggestion
            "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/note_taking#new_note_url",
```

**Review by Elchi3 — APPROVED** [review]

**Review by Earthman100 — COMMENTED** [review]
Commit

**Pull request merged**